openstack/octavia/templates/octavia-api-ingress.yaml

```diff
@@ -11,9 +11,10 @@ metadata:
     system: openstack
     type: api
     component: octavia
-{{- with .Values.ingress.annotations }}
   annotations:
   {{- include "utils.linkerd.pod_and_service_annotation" $ | indent 4 }}
+
+{{- with .Values.ingress.annotations }}  
 {{ toYaml . | indent 4 }}
```

Comment:
I'd rather not add an empty line here so it's easier to see that these belong to annotations. It would imho also make sense to indent the `with` and `end` block (and maybe the `toYaml`, too).

Additionally, with moving the `include` out of the `with`, you don't need to use `$` anymore.

Reply:
Fixed empty line and $/.
Re indentation, it is not rendering properly if indented, so I left it

Reply:
I pushed commit 573099c06a1cbe7136fb6fceff3ac54de8939491 to fix this.